packages/api/scripts/generate-upload-data.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * This script will generate many uploads for a given user token using the web3.storage API endpoints.
+ * Handy when working with and testing features that require a pristine mass set of data, such as pagination.
+ * Arguments can be passed with a arg=value pattern, ie token='<YOUR TOKEN HERE>'. Passing a token is the only required argument.
+ * Not passing a value will default to a truthy value. ie appending `pin` argument is the same as `pin=true`.
+ * This script requires a users token to be passed or assigned to an environment variable API_TOKEN.
+ * Passable arguments are:
+ * - token: the token of the user these uploads will be authenticated with.
+ * - uploads: The amount of uploads that need to be added, defaulting to 25.
+ * - url: The URL of the API, defaulting to localhost.
+ * - pin: Adds another request for each upload to pin that upload generating pin results, defaulting to false.
```

Comment:
I wonder if we can make this slightly clearer. 
As usual "pin" is a confusing word 😢 

I would try to make it extremely obvious that here we refer to creating Pinning Service API requests here rather than just saying pin, which might be confused with cluster pinning.

maybe something along these lines:

```
psa_pin_requests: If true, on top of uploading the file, a PSA pin request is created for the given CID. 
```

What do you think? 
Feel free to update/reword.

Reply:
Really good point! Yep this makes so much more sense, thanks